docs/codeql/codeql-language-guides/analyzing-data-flow-in-cpp.rst

```diff
@@ -1,5 +1,9 @@
 .. _analyzing-data-flow-in-cpp:
 
+.. pull-quote:: Note
+
+   The data flow library described here will be deprecated in the near future. For the replacement library see :ref:`here <analyzing-data-flow-in-cpp-new>`.
```

Comment:
I like "will be deprecated in the near future" as an advanced warning here, that doesn't push users too hard.

Reply:
The actual push will be to slap `deprecated` on everything in the AST dataflow library.